.travis.yml

```diff
@@ -32,7 +28,7 @@ _android_job_template: &android_job_template
       - $HOME/.gradle/wrapper/
   script:
     - cd android
-    - ./gradlew checkstyle assembleRelease test
+    - ./gradlew assembleRelease test
```

Comment:
What about just ./gradlew buildRelease? According to docs:
build: Assembles and tests this project.

Reply:
Nice catch!